packages/excalidraw/element/embeddable.ts

```diff
@@ -217,21 +212,6 @@ export const getEmbedLink = (
   return { link, intrinsicSize: aspectRatio, type };
 };
 
-export const isIframeLikeOrItsLabel = (
-  element: NonDeletedExcalidrawElement,
-): Boolean => {
-  if (isIframeLikeElement(element)) {
-    return true;
-  }
-  if (element.type === "text") {
-    const container = getContainerElement(element);
-    if (container && isFrameLikeElement(container)) {
-      return true;
-    }
-  }
-  return false;
-};
-
```

Comment:
removed as it doesn't seem to be used in the sense it did anything (we don't create frame labels as text elements, putting them in the renderable elements array/map — we create and render them on the fly)